mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBDialect.java

```diff
@@ -222,7 +224,7 @@ else if ( isInTheQueue( key, tupleContext ) ) {
 
 	private static boolean isInTheQueue(EntityKey key, TupleContext tupleContext) {
```

Comment:
They don't have the same parameters. But I should have renamed this one too. Will do.